src/main/java/ExeStreamAPI.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Comparator;
+import java.util.List;
+
+public class ExeStreamAPI {
+  public void indexOrderOut(List<String> list) {
+    System.out.printf("1.インデックス順表示：");
```

Comment:
下記でよいかと思います。
```suggestion
    System.out.print("1.インデックス順表示：");
```
printfは指定したフォーマットに対して引数を入れ込むときにつかいます。
https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/io/PrintStream.html#printf(java.lang.String,java.lang.Object...)
`java printf`で調べてみると良いです。